### PR TITLE
[DO NOT MERGE] Some more updates

### PR DESCRIPTION
**Pull request by johnmu**

### CODE REVIEW

**Comment by johnmu** [comment]
@marghoob can you check this? if it is ok feel free to merge

File: scripts/annotate_vcf_bam.py [hunk 1]
```diff
@@ -0,0 +1,203 @@
+#!/net/kodiak/volumes/lake/shared/users/marghoob/my_env/bin/python
```
Comment (marghoob) [inline scripts/annotate_vcf_bam.py]: replace path to marghoob by /usr/bin/env python

File: scripts/annotate_vcf_bam.py [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+                # get discordant reads stats
+                vcf_record.INFO["AA_DISCORDANT_HIGH"] = num_discordant_high
+                vcf_record.INFO["AA_DISCORDANT_LOW"] = num_discordant_low
+
+                # get supplementary alignment stats
+                # Skip this for now
+
+            vcf_writer.write_record(vcf_record)
+
+        vcf_writer.close()
```
Comment (marghoob) [inline scripts/annotate_vcf_bam.py]: close vcf_reader too.

File: scripts/annotate_vcf_bam.py (hunk 1 above)
Comment (johnmu) [inline scripts/annotate_vcf_bam.py]: uhh, so replace by 
`#!/usr/bin/env python`
?

File: scripts/annotate_vcf_bam.py (hunk 2 above)
Comment (johnmu) [inline scripts/annotate_vcf_bam.py]: it won't close :(

File: scripts/annotate_vcf_bam.py [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+    logging.basicConfig(level=logging.INFO, format=FORMAT)
+    logger = logging.getLogger(__name__)
+
+    parser = argparse.ArgumentParser(
+        description="Annotate VCF with additional useful features",
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument("--bam", help="BAM file", required=True, type=file)
+    parser.add_argument("--chromosomes", nargs="+", help="Chromosomes", default=[])
+    parser.add_argument("--workdir", help="Working directory", default="work")
+    parser.add_argument("--num_threads", help="Number of threads to use", default=1, type=int)
```
Comment (marghoob) [inline scripts/annotate_vcf_bam.py]: you don't need this option.

**Comment by marghoob** [comment]
looks good to me modulo some small suggestions.

File: scripts/annotate_vcf_bam.py (hunk 2 above)
Comment (marghoob) [inline scripts/annotate_vcf_bam.py]: you are right. what was i thinking. the iteration should take care of this.

On Thu, Feb 12, 2015 at 5:13 PM, John Mu <notifications@github.com> wrote:

> In scripts/annotate_vcf_bam.py
> <https://github.com/bioinform/metasv/pull/25#discussion_r24637294>:
>
> > +                # get clipped reads stats
> > +                if total_bases > 0.0:
> > +                    vcf_record.INFO["AA_PROP_ALIGNED"] = bases_aligned/total_bases
> > +                if end_total_bases > 0.0:
> > +                    vcf_record.INFO["AA_END_PROP_ALIGNED"] = end_bases_aligned/end_total_bases
> > +
> > +                # get discordant reads stats
> > +                vcf_record.INFO["AA_DISCORDANT_HIGH"] = num_discordant_high
> > +                vcf_record.INFO["AA_DISCORDANT_LOW"] = num_discordant_low
> > +
> > +                # get supplementary alignment stats
> > +                # Skip this for now
> > +
> > +            vcf_writer.write_record(vcf_record)
> > +
> > +        vcf_writer.close()
>
> it won't close :(
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bioinform/metasv/pull/25/files#r24637294>.
>



-- 

<http://www.bina.com>

*Marghoob Mohiyuddin, PhD  *//  Sr. Bioinformatics Scientist, Bina
Technologies
E. marghoob@bina.com

*Learn More*  //  video <http://www.bina.com/product.html>  //  news
<http://www.bina.com/press-coverage.html>  //  posters
<http://f1000.com/posters/search?query=bina+technologies>  //  science
<http://www.bina.com/science.html>  //  platform
<http://www.bina.com/platform.html>
*Test Drive Bina  //  Experience NextGen analysis for yourself *
<http://www.bina.com/test-drive.html>

**Pull request merged**